docs/user-guide/application-set.md

```diff
@@ -0,0 +1,43 @@
+### Automating the generation of Argo CD Applications with the ApplicationSet Controller
+
+Within the Argo CD project is the [ApplicationSet controller](https://github.com/argoproj-labs/applicationset) sub-project, which adds Application automation, improved multi-cluster support, and improved cluster multitenant support, to Argo CD.
+
```

Comment:
This is a very opinionated/subjective comment, but IMHO the first part of the sentence could be simplified to

> The ApplicationSet controller is a sub-project of Argo CD, which adds...

Reply:
It's not only about multi-cluster, it's also about developer independence. The git generator allows developers to create applications without having permission to the argocd namespace

Reply:
Thanks for your comments!

Re - subjectiveness, definitely @jannfis: the first sentence needs to be brief, and immediately inform the reader why they should be interested in the ApplicationsSet functionality (and thus keep reading the rest of the page). So the need for a "punchy" introductory sentence limits the degree to which nuance can be expressed on the strength of its argument.

Fortunately, I think we can change it to "*seeks to* improve multi-cluster support, (...)", implying that this is the goal of the controller rather than a universally agreed upon statement.

Re - developer independence, @OmerKahani: that was what the 'cluster multitenant support within Argo CD' phrase in the first sentence was referring to, but admitedly this was a bit opaque. I've added a more explicit reference as a second sentence. 

Anyways, let me know what y'all think about the new changes: it's definitely a balancing act between being concise, but still providing enough detail to concretely connect with the problems that the reader might be looking for solutions for.

Reply:
Going to merge it now - we can always further improve later. This is a great start to let more people know that ApplicationSet exists, and that Argo CD endorses it usage!